CHANGELOG.unreleased.md

```diff
@@ -12,6 +12,7 @@
 - [Plugin/auth-iodc] OpenID Connect scopes are now configurable and `profile` is included by default
 - [Dashboard/Health] Button to copy UUID of an orphan VDI to the clipboard (PR [#6893](https://github.com/vatesfr/xen-orchestra/pull/6893))
 - [Kubernetes recipe] Add the possibility to choose the version for the cluster [#6842](https://github.com/vatesfr/xen-orchestra/issues/6842) (PR [#6880](https://github.com/vatesfr/xen-orchestra/pull/6880))
+- [Backups] Add a setting to limit the number of disk transferred in parallel per VM (PR [#6787](https://github.com/vatesfr/xen-orchestra/pull/6787))
```

Comment:
that is why I think the configuration variable should be in the configuration file, it would be mostly auto documenting

nerveless, here is the hange